legal-api/src/legal_api/models/db.py

```diff
@@ -11,17 +11,222 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Create SQLAlchenmy and Schema managers.
+"""Create SQLAlchemy and Schema managers.
 
 These will get initialized by the application using the models
 """
+
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy_continuum import make_versioned
+from sqlalchemy.ext.declarative import declarative_base
+from sql_versioning import TransactionManager as NewTransactionManager
+from sqlalchemy.orm import sessionmaker, configure_mappers
+from sqlalchemy import event, Column, DateTime, func, Integer
+from sqlalchemy_continuum import make_versioned, versioning_manager
+from sqlalchemy.orm.session import Session as SQLAlchemySession
+
+Base = declarative_base()
 
+class Transaction(Base):
+    __tablename__ = 'transaction'
```

Comment:
`Tranasction` table can probably be removed.  Just left over code when experimenting.  Should be able to keep using sqlcontinuums underlying implementation for transaction model